packages/server/src/api/controllers/query/index.ts

```diff
@@ -169,8 +169,10 @@ function enrichParameters(
   validateQueryInputs(requestParameters)
   // make sure parameters are fully enriched with defaults
   for (const parameter of query.parameters) {
-    let value: string | null =
-      requestParameters[parameter.name] || parameter.default
```

Comment:
Can you use `??` here instead?

Reply:
You probably could now I think of it. Not going to go in and change, but a good spot.